### DISCUSSION TITLE
Copy source folders

### DISCUSSION THREAD

**Discussion by jose1711** [borgbase/vorta#983] [Ideas]
It's great that `vorta` has a paste folders/files function but how about a copy one? Like.. you want to make a profile similar to the current one but with a slightly different set of folders/files. Of course you can still get the data using sqlite but getting that via GUI would be very welcome. Currently if you select files and do copy-paste, only the last entry gets copied.

**Comment by m3nu** [comment]
I wouldnt add a new button for this, but maybe one could override the system copy key to include the full table.

**Reply by jose1711** [reply]
> I wouldnt add a new button for this, but maybe one could override the system copy key to include the full table.

Oh sure, that would be more than enough.
